src/bridges/liquity/StakingBridge.sol

```diff
@@ -43,6 +41,10 @@ contract StakingBridge is BridgeBase, ERC20("StakingBridge", "SB") {
     // Optimization based on EIP-1087
     uint256 internal constant DUST = 1;
 
+    // Smallest amounts of rewards to swap (gas optimizations)
+    uint256 private constant MIN_LUSD_SWAP_AMT = 1e20; // 100 LUSD
```

Comment:
I have just set them to what I consider reasonable values but I planned on discussing them with you. I think the values should be set in such a way that they are as high as possible without users caring too much. This is because when the swap doesn't execute it's an overall gain in value for the bridge depositors. For this reason I think it makes sense to have them higher than what is the value of expected gas costs of the swap. I would actually consider increasing them a bit in the stablecoin cases. In the non-stablecoin cases it's a bit more tricky because market movements affect it.